flink-connectors/flink-connector-kafka-0.8/src/test/java/org/apache/flink/streaming/connectors/kafka/Kafka08ProducerITCase.java

```diff
@@ -18,17 +18,19 @@
 
 package org.apache.flink.streaming.connectors.kafka;
 
-import org.junit.Test;
-
 /**
  * IT cases for the {@link FlinkKafkaProducer08}.
  */
 @SuppressWarnings("serial")
 public class Kafka08ProducerITCase extends KafkaProducerTestBase {
 
-	@Test
-	public void testCustomPartitioning() {
-		runCustomPartitioningTest();
+	@Override
+	public void testOneToOneAtLeastOnceRegularSink() throws Exception {
+		// TODO: enable this for Kafka 0.8 - now it hangs indefinitely
 	}
 
+	@Override
+	public void testOneToOneAtLeastOnceCustomOperator() throws Exception {
+		// Disable this test since FlinkKafka08Producer doesn't support writing timestamps
```

Comment:
I would perhaps rephrase this comment a bit:
it's disabled because FlinkKafka08Producer doesn't run in the custom operator mode (to be coherent with the test case name)